fighthealthinsurance/utils.py

```
@@ -380,6 +383,11 @@
             if required_tasks_finished >= len(required):
                 logger.debug("All done with required tasks")
                 break
+    except asyncio.TimeoutError as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)